.github/workflows/codespaces-test.yml

```
@@ -0,0 +1,73 @@
+      env:
+        POSTGRES_HOST: localhost
+        REDIS_HOST: localhost
+      run: |
+        python manage.py test
+    
+    - name: Check Django settings
+      env:
+        POSTGRES_HOST: localhost
+        REDIS_HOST: localhost
+      run: |
+        python manage.py check --deploy
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)